auth/builtin.go

```diff
@@ -59,13 +59,28 @@ func (r *NoAuth) Refresh(refresh string) (token Token, err error) {
 }
 
 // Builtin auth provider.
-type Builtin struct {
+type Builtin struct{}
+
+// Parse Token out of a string
+func ParseToken(requestToken string) (string, error) {
+	splitToken := strings.Split(requestToken, " ")
+	if splitToken[0] != "Bearer" {
+		return "", errors.New("authentication header not of type bearer")
+	}
+	token := strings.TrimSpace(splitToken[1])
```

Comment:
This will result in a SEGV when requestToken=`Bearer`. no token.

parseToken() should return `NotValid` which will result in the request http=403 (forbidden) instead of 500.